src/remotestorage.js

```diff
@@ -12,6 +12,7 @@ const log = require('./log');
 const Features = require('./features');
 const globalContext = util.getGlobalContext();
 const eventHandling = require('./eventhandling');
+const getJSONFromLocalStorage = util.getJSONFromLocalStorage;
```

Comment:
As we have to import all of `util` right now, growing it more and more seems like it's not a great idea. What do you think about making this its own import?

Reply:
I agree in general, but this file needs to import all of `util` anyway, as it's setting it as `RemoteStorage.util` in [line 773](https://github.com/remotestorage/remotestorage.js/blob/refactorings/src/remotestorage.js#L773).